### DISCUSSION TITLE
corsixth: ffmpeg 7 unsupported

### DISCUSSION THREAD

**Discussion by lewri** [Homebrew/discussions#5361] [Everyday usage]
### Output of `brew config`

```shell
/
```

### Output of `brew doctor`

```shell
/
```

### Description of issue

Hello! CorsixTH dev here.
Recently ffmpeg 7 was released. The CorsixTH homebrew formulae was updated to point to this version.
https://github.com/Homebrew/homebrew-core/blob/0388385c86d49b1127d0ae22246e632e0520dadd/Formula/c/corsixth.rb

However, we have no certified support yet for ffmpeg 7 and would recommend rolling back to 5.1 for now. As I don't know homebrew's workflow (or own a Mac) I'm unable to submit a PR to fix this.

Thought someone would like to know though!
PS: CorsixTH head also has no ffmpeg 7 support either, but it should be in place for the next version.

**Comment by SMillerDev** [comment]
We don't really care about certified support. The tests pass, so it seems to work as expected and that's good enough for Homebrew. If people complain that it doesn't work we can consider rolling back to 6.x.

**Reply by lewri** [reply]
Heh, funny enough we don't even support 6.x yet -- but my local builds work fine on it 😅
